pageserver/src/tenant/layer_map/historic_layer_coverage.rs

```diff
@@ -548,25 +607,32 @@ fn test_retroactive_simple() {
         LayerKey {
             key: 2..5,
             lsn: 105..106,
-            is_image: true,
+            is_image: false,
         },
         "Delta 1".to_string(),
     );
 
     // Rebuild so we can start querying
     map.rebuild();
 
-    // Query key 4
-    let version = map.get().unwrap().get_version(90);
-    assert!(version.is_none());
-    let version = map.get().unwrap().get_version(102).unwrap();
-    assert_eq!(version.image_coverage.query(4), Some("Image 1".to_string()));
-    let version = map.get().unwrap().get_version(107).unwrap();
-    assert_eq!(version.image_coverage.query(4), Some("Delta 1".to_string()));
-    let version = map.get().unwrap().get_version(115).unwrap();
-    assert_eq!(version.image_coverage.query(4), Some("Image 2".to_string()));
-    let version = map.get().unwrap().get_version(125).unwrap();
-    assert_eq!(version.image_coverage.query(4), Some("Image 3".to_string()));
+    {
+        let map = map.get().expect("rebuilt");
```

Comment:
This refactor causes distracting churn in this diff.
IIUC the only thing that actually changes was the addition of

```
        assert_eq!(version.delta_coverage.query(4), Some("Delta 1".to_string()));
```

below, correct?

Reply:
Yes, it's in separate commit from the `map.get().expect("rebuilt")` changes.

Reply:
Ok, would be nice if you can keep them separate when merging, or merge that commit as an independent PR

Reply:
I don't think that it's that important, it's a minor test-misunderstanding after all. While I agree the follow up refactoring make the both a bit messy when looking at the first time.